doc/reference/networking/net_stats.rst

```diff
@@ -6,6 +6,25 @@ Network Statistics
 Overview
 ********
 
+Network statistics is collected if :option:`CONFIG_NET_STATISTICS` is set.
+Individual component statistics like for IPv4 or IPv6, can be turned off
+if those statistics are not needed. See various options in
+:file:`subsys/net/ip/Kconfig.stats` file for details.
+
+By default system collects network statistics per network interface. This
+can be controlled by :option:`CONFIG_NET_STATISTICS_PER_INTERFACE` option.
+
+The :option:`CONFIG_NET_STATISTICS_USER_API` option can be set if the
+application wants to collect statistics for further processing. The network
+management interface API is used for that. See :ref:`net_mgmt_interface` for
+details.
+
+If the :option:`CONFIG_NET_STATISTICS_ETHERNET_VENDOR` option is set, then
```

Comment:
Thanks for all the docs improvements. But why do we jump to CONFIG_NET_STATISTICS_ETHERNET_VENDOR here? Please first say that CONFIG_NET_STATISTICS_ETHERNET exists for generic Ethernet stats, and beyond that,  there's CONFIG_NET_STATISTICS_ETHERNET_VENDOR for additional stuff.

Reply:
I will add a comment about generic Ethernet stats.